frontend/src/views/components/Journey/Journey.component.tsx

```diff
@@ -103,31 +103,38 @@ const Journey: FC<JourneyProps> = ({
   useEffect(() => {
     if (!journeyLoaded && currentJourney) loadJourney(currentJourney);
   }, [currentJourney, journeyLoaded, loadJourney]);
-
-  const { steps } = journey;
   const stepIndex = Number(step) - 1;
+  const { steps, attributes: journeyAttributes } = journey;
+  const { published } = journeyAttributes || {};
 
   if (!journeyLoaded || !journey || !steps[stepIndex]) return null;
   const stepInfo = steps[stepIndex];
 
   const { attributes } = stepInfo;
   const { step_type: stepType } = attributes;
   return (
-    <div className="l-journey" id="journeyIndexView">
-      <Loader loading={journeyLoading} />
-
-      {journeyLoaded && React.createElement(JOURNEY_TYPES[stepType], { ...attributes })}
-
-      <Controls journeysLength={journeysLength} slideslength={steps.length} />
-
-      {!journeyLoading && stepType !== 'embed' && (
-        <p className={`credits ${stepType}`}>
-          <a target="_blank" rel="noopener noreferrer" href={attributes.credits_url}>
-            {attributes.credits}
-          </a>
-        </p>
-      )}
-    </div>
+    <>
+      <Head>
+        {published === false && (
+          <meta name="robots" content="noindex, nofollow, noimageindex, noarchive" />
+        )}
```

Comment:
Is better if this is checked as false. Published could be undefined if the journey is not loaded yet

Reply:
No, I want to render it only when is false

Reply:
We should test this because I'm not sure about the search bots will be able to read it before being indexed. If we only add the `nofollow` when the journey has published false it should be decided in the server.

Reply:
Yes, maybe it has to come from the server. I wanted to test it because the content is not available until we have this published attribute